classes/ezjsctags.php

```diff
@@ -19,15 +19,26 @@ static public function autocomplete( $args )
         $http = eZHTTPTool::instance();
 
         $searchString = trim( $http->postVariable( 'search_string' ), '' );
+        $broadMatch = eZINI::instance('eztags.ini')->variable( 'GeneralSettings', 'BroadMatchAutoComplete' )  == 'enabled';
+
         if ( empty( $searchString ) )
             return array( 'status' => 'success', 'message' => '', 'tags' => array() );
 
         // Initialize transformation system
         $trans = eZCharTransform::instance();
         $searchString = $trans->transformByGroup( $http->postVariable( 'search_string' ), 'lowercase' );
+        $searchStringParam = '';
```

Comment:
This could be simplified to:

```
$searchStringParam = $searchString . '%';
if ($broadMatch)
{
    $searchStringParam = '%' . $searchStringParam;
}
```